cli/tests/pcluster/config/test_validators.py

```diff
@@ -185,6 +185,24 @@ def test_ec2_ami_validator(mocker, boto3_stubber, image_architecture, bad_ami_me
     utils.assert_param_validator(mocker, config_parser_dict, expected_message)
 
 
+@pytest.mark.parametrize(
+    "section_dict, expected_message",
+    [
+        ({"tags": {"key": "value", "key2": "value2"}}, None),
+        (
+            {"tags": {"key": "value", "Version": "value2"}},
+            'You are using "Version" as a key in your tags. Because "Version" tag is reserved to '
```

Comment:
In the test is not needed to use the entire message, you can use a substring. It simplify the work if we need to do minor changes in the message.

We just need to be sure that is failing.